crates/core/src/database.rs

```diff
@@ -19,12 +22,36 @@ impl Database {
     }
 
     pub fn new(url: String) -> Database {
-        Database { url }
+        Database {
+            url,
+            replicator: None,
+        }
+    }
+
+    pub fn with_replicator(url: impl Into<String>) -> Database {
+        Database {
+            url: url.into(),
+            // FIXME: we probably shouldn't hardcode this part
+            replicator: Some(Replicator::new("data.libsql").unwrap()),
```

Comment:
even better - nuked it :innocent: we'll need to push the code where to put our intermediary metadata somewhere else, so that we don't assume any directory structure (which slqd has)